third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -4944,6 +4945,10 @@ void SetupRecordReplayCommands(v8::Isolate* isolate, LocalFrame* localFrame) {
     RunScript(isolate, context, gSourceMapScript, InternalScriptURL);
   }
 
+  // Call this here to avoid divergence later.
+  // https://linear.app/replay/issue/RUN-2195#comment-e0b6c75b
+  localFrame->GetSettings()->SetForceMainWorldInitialization(true);
```

Comment:
This should be gated on a feature test so we can easily disable it to see the performance impact.

Reply:
I forgot yet again - thanks! 👀